templates/server/src/main/resources/archetype-resources/api/pom.xml

```diff
@@ -13,6 +13,11 @@
   <description>API of the server for the ${rootArtifactId} application (containing datatypes, transfer-objects, and service interfaces).</description>
 
   <dependencies>
+    <dependency>
+      <groupId>org.springframework.data</groupId>
+      <artifactId>spring-data-commons</artifactId>
+      <version>2.1.2.RELEASE</version>
```

Comment:
Version already comes from BOM and should not be specified redundantly here as we might miss that on upgrades and have additional maintenance effort.